image-layout.md

```diff
@@ -115,4 +116,42 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## Index 
+
+Index is intended to be generated from the content in an image-layout after any updates have been made.
+Allows for mapping random access to the references in the image-layout, and needing only to read from the `oci-layout` once to build this map of references.
+
+**index** *array of objects*, OPTIONAL
+
+  Provides a single access to top-level information about the layout.
```

Comment:
If you want to punt for now, that's fine with me (I'm not a consumer of the compiled HTML or PDF output).  But runtime-spec has had both opencontainers/runtime-spec#495 and opencontainers/runtime-spec#608 to avoid issues with two-space indents for sub-lists.  It seems like we should be sticking to that pattern here for new additions, and eventually someone will be bothered enough [by][1]:

```html
<ul>
<li><strong><code>mediaType</code></strong> <em>string</em></li>
</ul>
<p>This REQUIRED property contains the MIME type of the referenced content.</p>
<p>The OCI image specification defines <a href="media-types.md">several of its \
own MIME types</a> for resources defined in the specification.</p>
```

That they'll go through and fix the existing Markdown ;).

[1]: https://github.com/opencontainers/image-spec/releases/download/v1.0.0-rc2/oci-image-spec-v1.0.0-rc2.html